Makefile

```diff
@@ -108,3 +108,11 @@ stop-prometheus-alertmanager:
 .PHONY: start-node-exporter
 start-node-exporter:
 	./prometheus/scripts/start-node-exporter.sh --web.listen-address 0.0.0.0:10080
+
+.PHONY: runtest
+runtest: ./local-test-repo/.git
```

Comment:
Why do we need this make target, when running `dune runtest` also works :)

Reply:
`dune runtest` would run the tests on the host machine. I'd like to run tests in the `pipeline` docker container and not rely on having a local opam setup. Does that make sense or am I missing some thing?

Reply:
Ah yes. I would install the dependencies for running the test in my local switch rather than using the make command. But maybe the other workflow can also be used by other people? I don't have strong opinions on this :)

Reply:
I guess it does help with editor support, etc., to have the local opam switch updated with all the dependencies. But, it could also seem like extra work to keep the local switch updated with all the deps. May be it's just that I'm still not super comfortable with OCaml and related tools, that I prefer running the tests in the docker container which is guaranteed to have all the required deps for a particular commit that I'm running tests on :)